src/components/NewBook.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Card from './UI/Card';
+
+const NewBook = () => (
+  <Card>
+    <h3>Add New Book</h3>
+    <form>
+      <div>
+        <input type="text" placeholder="Book Title" />
```

Comment:
- [ ] Nice one adding an input option for `book` title, kindly note that
  - [ ] The `input` tag doesn't have the `name` attribute.
  - [ ] The `form` doesn't have an `author` input, which is also a requirement.

Reply:
Hi @AdnanOlarmmi ,

Thanks for your time and for reviewing my code. I have updated my code as you advised.